backend/geonature/migrations/versions/0e8e1943c215_add_missing_error.py

```diff
@@ -0,0 +1,34 @@
+"""add_missing_error
```

Comment:
```suggestion
"""add_import_missing_boolean_error
```
Et renommer aussi le fichier de migration ?

Reply:
fait dans #3077